src/main/java/yevhen/bookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -0,0 +1,44 @@
+package yevhen.bookstore.exception;
+
+import java.time.LocalDateTime;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.validation.ObjectError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class CustomGlobalExceptionHandler extends ResponseEntityExceptionHandler {
+    @Override
+    protected ResponseEntity<Object> handleMethodArgumentNotValid(
+            MethodArgumentNotValidException ex,
+            HttpHeaders headers,
+            HttpStatusCode status,
+            WebRequest request) {
+        Map<String, Object> errorMessage = new LinkedHashMap<>();
+        errorMessage.put("timestamp", LocalDateTime.now());
+        errorMessage.put("status", HttpStatus.BAD_REQUEST);
```

Comment:
errorMessage.put("timestamp", LocalDateTime.now());
errorMessage.put("status", HttpStatus.BAD_REQUEST);
we don't need it